### PR TITLE
update filter product by ajax and fix route in js file

### PR DESCRIPTION
**Pull request by thangdangminh**
Add new image of products

### CODE REVIEW

**Comment by thangdangminh** [comment]
ready

File: resources/js/app.js [hunk 1]
```diff
@@ -104,6 +81,55 @@ $(document).ready(function () {
...
+        "<div class='card-body'><h4 class='card-title'>" +
+                "<a href='" + route('shop.show', {productSlug: response.products[i].slug}) +"'>" +
+                response.products[i].name + "</a></h4><h5>" +
+                response.products[i].price + "</h5>" +
+            "<p class='card-text'>" + response.products[i].description +"</p></div>"+
+        "<div class='card-footer'>" +
+            "<button type='button' data-qty='" + response.products[i].stock_quantity + "' " +
+            "data-slug='"+ response.products[i].slug +"' " +
+            "class='add-product btn btn-outline-primary float-right " + checkDisabilityBtn + "'>"
+            + "Buy now</button></div></div></div>"
```
Comment (tranducquoc) [inline resources/js/app.js]: i18n js

File: resources/views/shop/index.blade.php [hunk 2]
```diff
@@ -27,25 +27,20 @@
...
             <div class="list-group">
-                {!! Form::open([
-                    'method' => 'get',
-                    'route' => 'shop.filter.price',
-                ]) !!}
-                    {!! Form::text('price_range', '', ['class' => 'js-range-slider']) !!}
-                    {!! Form::submit(@trans('common.form.button.submit'), ['class' => 'btn btn-outline-primary']) !!}
-                {!! Form::close() !!}
+                <input type="text" name="price_range" class="js-range-slider">
+                <button type="btn" class="filter-price-btn btn btn-primary">Submit</button>
```
Comment (tranducquoc) [inline resources/views/shop/index.blade.php]: i18n

**Comment by thangdangminh** [comment]
ready

**Comment by thangdangminh** [comment]
ready

**Pull request merged**